File: lib/src/main/java/com/example/lib/java/study/TimeStampTest.java
package com.example.lib.java.study;

import com.example.lib.java.util.LoggingLogger;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.TimeZone;

public class TimeStampTest{


    public static void main(String[] args){
        LoggingLogger.getInstance().print("cn time " + getBeijingDatetime());
    }

    private static String getBeijingDatetime() {
        @SuppressWarnings("SimpleDateFormat")
        DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZ");
        TimeZone tz = TimeZone.getTimeZone("Asia/Shanghai");
        df.setTimeZone(tz);
        Date date = new Date();
        return df.format(date);
    }
}

File: TestModule/src/test/kotlin/com/example/lib/basic/TimeStampTest.kt
package com.example.lib.basic

import com.example.lib.java.util.CustomLogger
import org.junit.jupiter.api.Test
import java.text.DateFormat
import java.text.SimpleDateFormat
import java.util.*

class TimeStampTest {
    private fun getBeijingDatetime(): String? {
        val df: DateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZ")
        val tz = TimeZone.getTimeZone("Asia/Shanghai")
        df.timeZone = tz
        val date = Date()
        return df.format(date)
    }

    @Test
    fun `베이징 시간 확인하기`() {
        CustomLogger.getInstance().info("cn time " + getBeijingDatetime())
    }
}